docs/python-grader/index.md

```diff
@@ -191,6 +191,8 @@ The code feedback library contains built-in functions for checking correctness o
   Checks that a numpy array has the same shape and datatype as the reference solution, also checks values to see if they are close using specified `rtol` and `atol`.
 - `check_list(name, ref, data, entry_type=None)`
   Checks that a list has the same length as the reference solution. If `entry_type` is not `None`, can optionally check if each element has that type. Does _not_ check values against the reference.
+- `check_dict(name, ref, data, *, partial_keys=None, check_keys=False, check_values=False, key_type=None, value_type=None, accuracy_critical=False, report_failure=False)`
```

Comment:
This has parameters that were removed.

Reply:
Fixed in b0154849a9187cec010fdefb137a06043532db88.